documentation/contributing/future_projects/index.markdown

```diff
@@ -114,6 +114,14 @@ For this, we need to identify and implement standard tasks at varying levels of
 - **Potential mentors**: Andy Zelenak, Henning Kayser
 - **Description**: There are several Inverse Kinematics libraries that need to be ported to MoveIt 2. Prime candidates are [BioIK](https://github.com/TAMS-Group/bio_ik) and [TRAC-IK](https://bitbucket.org/traclabs/trac_ik/src/master/).
 
+## MoveIt Wrapper for Reinforcement Learning
+
+- **Prerequisites**: ROS, MoveIt, reinforcement learning
+- **Programming skills**: C++
+- **Difficulty**: Medium
+- **Potential mentors**: Andy Zelenak, Mark Moll
+- **Description**: Conduct a brief survey of reinforcement learning methods that could be implemented over the summer. Choose one and implement it. One possibility is described in "Operator training for preferred manipulator trajectories" by Sharp, Horn, & Pryor.
```

Comment:
I like the idea of doing something with reinforcement learning and MoveIt. However, I'd prefer to use existing implementations of RL algorithms, and focus more on creating the right interface between RL implementations and MoveIt. Can you have a look at this work and see how that could be used for a potential GSoC project?:
https://arxiv.org/abs/1903.06282
https://arxiv.org/abs/1903.06278

Reply:
A "survey" is a really open ended task that most interns will get lost in. What if an intern just extends / hardens / documents better what @bostoncleek already started last summer with DexNet RL grasping?

Note the GSoC program is super handicaped now, with much less time to accomplish stuff during the summer. @mamoll can explain more.

Reply:
@davetcoleman Last summer I used GPD and DexNet. It would be great if an intern could improve the previous work. I think the intern would benefit if they were given a clear goal of what is expected from them by the end of the internship.

Reply:
It's not quite an internship: 10 weeks at 18 hours per week. That's why it cannot be too open-ended. Having Boston's work as concrete starting point helps, but I agree with Boston that a specific end goal would also be helpful. OTOH, leaving it somewhat vague and forcing the applicant to lay out a development roadmap (which is part of their application), would perhaps make it easier to select the better candidates :-).

Reply:
I should have been more clear -- this isn't grasp planning reinforcement learning. It's trajectory shaping. Of course there are a lot of things you can do with reinforcement learning.

I agree that hardening Boston's previous work is more important than starting something new.

Reply:
@bostoncleek can you add a new section to this page for extending your grasp work?
https://moveit.ros.org/documentation/contributing/future_projects/

Reply:
@davetcoleman yes, updating now.